pkg/formatted/k8s.go

```diff
@@ -65,17 +66,18 @@ func Condition(c v1beta1.Conditions) string {
 	case corev1.ConditionUnknown:
 		status = "Running"
 	}
-	cstatus := ColorStatus(status)
 
 	if c[0].Reason != "" && c[0].Reason != status {
-		if c[0].Reason == "PipelineRunCancelled" || c[0].Reason == "TaskRunCancelled" {
-			status = ColorStatus("Cancelled") + "(" + c[0].Reason + ")"
-		} else if c[0].Reason != status {
-			status = cstatus + "(" + c[0].Reason + ")"
+		switch c[0].Reason {
+		case "PipelineRunCancelled", "TaskRunCancelled":
+			return ColorStatus("Cancelled") + "(" + c[0].Reason + ")"
+		case "PipelineRunStopping", "TaskRunStopping":
+			return ColorStatus("Failed") + "(" + c[0].Reason + ")"
+		case "CreateContainerConfigError", "ExceededNodeResources", "ExceededResourceQuota":
```

Comment:
I am also wondering about the `Pending` status. Should this be an Error instead?

Reply:
Correct me if wrong, I put the Pending status considering that these can be rescheduled later. Controller mark them as Running cc @vdemeester

Reply:
Ah, makes sense. Sorry, I was thinking of a specific case with `CreateContainerConfigError` where containers are running as root in restricted environments, but that could be fixed as well and rescheduled. I think that makes sense then.